app_spec/test/files/links.js

```diff
@@ -1,6 +1,32 @@
-const { one, two } = require('../config')
+const { one, twoSame } = require('../config')
+const { Config } = require('@holochain/tryorama')
 
 module.exports = scenario => {
+
+  scenario.only('links propagate within a single conductor', async (s, t) => {
+    const { alice } = await s.players({alice: twoSame}, true)
+
+    await alice.call('app1', 'simple', 'create_link',
+      { base: alice.info('app1').agentAddress, target: 'Posty' }
+    )
+
+    await s.consistency()
+
+    const posts1 = await alice.call('app2', 'simple', 'get_my_links',
+      { base: alice.info('app2').agentAddress, status_request: 'Live' }
+    )
+
+    const posts2 = await alice.call('app2', 'simple', 'get_my_links',
+      { base: alice.info('app2').agentAddress, status_request: 'Live' }
```

Comment:
Hey just trying to figure out what is happening because I wanted to take a look at it. 

Should this be alice.info('app1').agentAddress? only asking because at this point we wont receieve any results from posts1 or posts2 because the link was created with a different agentAddress

Reply:
omg...yes it should be. I just fixed it, let's see what the test says now..